include/core/simulation_control.h

```diff
@@ -100,6 +100,11 @@ class SimulationControl
   // by the time
   double output_time_frequency;
 
+  // Output window
+  // Output file will be generated if current step number is in the output time
```

Comment:
The phrasing is a bit confusing. I'd say that this adds another condition for the output (in addition to the step respecting the output frequency parameter).